validation-test/stdlib/ArrayNew.swift.gyb

```diff
@@ -542,7 +542,7 @@ ArrayTestSuite.test("BridgedToObjC/Verbatim/objectAtIndex") {
   expectEqual(30, (v as! TestObjCValueTy).value)
   let idValue2 = unsafeBitCast(v, to: UInt.self)
 
-  for i in 0..<3 {
+  for _ in 0..<3 {
```

Comment:
Pretty sure this test is buggy, and the intention was to actually loop over something or not to loop at all...

Reply:
Yes, I think it can be removed.